core/src/identity.rs

```diff
@@ -29,7 +29,7 @@ pub mod secp256k1;
 pub mod error;
 
 use self::error::*;
-use crate::{PeerId, keys_proto};
```

Comment:
I am guessing that the changes in `identity.rs` and `core/src/identity/error.rs` were done by rustfmt. Would you mind reverting them in this pull-request as they seem unrelated? If you feel like the suggestions would improve code readability please open a separate pull request.

Reply:
Yep, editor has it in by default. I usually catch these, seems I missed one.